frontend/src/components/common/authorized-menu/styles.module.scss

```diff
@@ -4,7 +4,7 @@
   display: flex;
   flex-direction: column;
   min-width: 270px;
-  height: 89.6vh;
+  height: 100vh;
```

Comment:
But the total height will be more, than screen height, because we also have a header. And the screen ( including sidebar ) will be scrollable, even if we don't have enough content